VERSION

```diff
@@ -1 +1 @@
-v0.3.0-dev
+v0.3.0
```

Comment:
We may want to change VERSION to v0.4.0-dev when merging to main while v0.3.0 when merging to release-0.3?

Reply:
Yes, the purpose is we need to change it to v0.3.0 for release-0.3 branch. so either we change it in main now or we change it in the new branch.